jmx/jmx.go

```diff
@@ -323,6 +323,7 @@ func doQuery(ctx context.Context, out chan []byte, queryErrC chan error, querySt
 			queryErrC <- fmt.Errorf("reading nrjmx stdout: %s", err.Error())
 		}
 		out <- b
+		return
```

Comment:
I do not understand the purpose of the `for` loop. Maybe I'm mistaken, but don't all execution paths lead to the loop breaking on the first iteration?

Reply:
yes, you are right. As I understood, it was kept in hope to read all output from nrjmx, but it didn't happen